doc/STREAMING_ITERABLES.md

```diff
@@ -7,11 +7,144 @@
 - [Iterable Streams](#iterable-streams)
   - [Table of Contents](#table-of-contents)
   - [Usage Guide](#usage-guide)
+    - [Wrapping Duplex Iterables](#wrapping-duplex-iterables)
+  - [Iterable Stream Types](#iterable-stream-types)
+    - [Source](#source)
+    - [Sink](#sink)
+    - [Transform](#transform)
+    - [Duplex](#duplex)
   - [Iterable Modules](#iterable-modules)
 
 ## Usage Guide
 
-**Coming Soon!**
+### Wrapping Duplex Iterables
+
+Sometimes you may need to wrap an existing duplex stream in order to perform incoming and outgoing [transforms](#transform) on data. This type of wrapping is commonly used in encryption streams. Using [it-pair][it-pair] and [it-pipe], we can do this rather easily, given an existing [duplex iterable](#duplex).
+
+```js
+const duplexPair = require('it-pair/duplex')
+const pipe = require('it-pipe')
+
+// Wrapper is what we will write and read from
+// This gives us two duplex iterables that are internally connected
+const wrapper = duplexPair()
```

Comment:
You might want to name these pairs to make it easier to understand in the example.

```suggestion
const [client, server] = duplexPair()
```

Reply:
I went with `internal` and `external`. I don't feel great about these names, but the pairing is used to have 1 half integrate with the external duplex connection and the other half is what we would use internally to read/write data to.